src/main/java/com/faforever/client/fa/ForgedAllianceService.java

```diff
@@ -36,6 +36,18 @@ public ForgedAllianceService(PreferencesService preferencesService) {
     this.preferencesService = preferencesService;
   }
 
+  public Process startGameOffline(List<String> args) throws IOException {
+    Path executable = getExecutable();
+    List<String> launchCommand = LaunchCommandBuilder.create()
+        .executableDecorator(preferencesService.getPreferences().getForgedAlliance().getExecutableDecorator())
+        .executable(executable)
+        .additionalArgs(args)
+        .logFile(preferencesService.getFafLogDirectory().resolve("game.log"))
+
+        .build();
```

Comment:
So its not possible to have replay for playing tutorials for now? Why not?

Reply:
i asked @Speed2 on Zulip

Reply:
it produces a local replay anyway even if u dont not register a replay server 
![image](https://user-images.githubusercontent.com/19996713/67492360-98b3ba00-f676-11e9-8501-340a999f707d.png)

Reply:
in the documents folder

Reply:
it is not that easy to start because it misses out on some info for example what featured mod is needed etc. but would be sufficent for debugging.

Reply:
u would need to start game by hand to watch :D

Reply:
So what know? I would say that is sufficent...